src/main/java/org/eolang/sodg/Depo.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+ * SPDX-License-Identifier: MIT
+ */
+package org.eolang.sodg;
+
+import com.yegor256.xsline.Shift;
+import com.yegor256.xsline.Train;
+
+/**
+ * Train depo.
+ * @since 0.0.3
+ */
+interface Depo {
```

Comment:
@h1alexbel there is no such thing as "depo" in eng. Did you mean [depot](https://dictionary.cambridge.org/dictionary/english/depot)? Some javadocs would greatly help understand the purpose of this abstraction.